src/pal/inc/pal.h

```diff
@@ -4618,6 +4618,9 @@ PALIMPORT DLLEXPORT int __cdecl _putenv(const char *);
 
 #define ERANGE          34
 
+PALIMPORT WCHAR __cdecl ToUpperInvariant(WCHAR);
```

Comment:
For PAL functions that are not equivalents of Windows APIs, the conventions is to use PAL_ prefix. So this should be PAL_ToUpperInvariant.